modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -7,6 +7,8 @@
 [abstract]
 The Couchbase Scala SDK enables you to interact with a Couchbase Server cluster from the Scala language.
 
+Our new Scala SDK offers functional programming-friendly ways of working with Couchbase; interoperability with the Java SDK; first class integration with the _akka_ stack; and native _Spark_ integration.
```

Comment:
Wellll not really on the last two...  We don't touch Akka and Spark is provided by the Spark Connector (which uses the Java SDK).  Also not too much interop with the Java SDK.

Reply:
So is it just "Our new Scala SDK offers functional programming-friendly ways of working with Couchbase." ?

(Which we might as well omit/integrate into the `[abstract]` above...)